package.json

```diff
@@ -13,7 +16,8 @@
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
     "socket.io-client": "^2.2.0",
-    "styled-components": "^4.2.0"
+    "styled-components": "^4.2.0",
+    "uid": "^0.0.2"
```

Comment:
What is this? If it is a ID generator, why do we need it here?

Reply:
deleteed